pettingzoo/utils/env.py

```diff
@@ -3,12 +3,12 @@
 import warnings
 from typing import Any, Dict, Iterable, Iterator, List, Optional, Tuple, TypeVar
 
-import gym
-import numpy as np
+import gym  # type: ignore
+import numpy as np  # type: ignore
```

Comment:
I would have expected the linter to get the types from both Gym and NumPy, not so sure why these need to be ignored... :thinking:

Reply:
I believe adding this `reportMissingTypeStubs = false` to the pyproject.toml should fix this.

Reply:
Yes it does, thanks!

Reply:
Well apparently it does on my machine but not in CI... :thinking:

Reply:
What exactly is the issue that requires this?

Reply:
Not sure what happened exactly but now it works with @pseudo-rnd-thoughts suggestion, so I removed it.